examples/hooks-next/pages/index.tsx

```diff
@@ -58,12 +60,31 @@ export default function HomePage({ serverState, url }: HomePageProps) {
         }}
       >
         <SearchBox />
+        <DynamicWidgets fallbackComponent={FallbackComponent} />
```

Comment:
should I remove the dynamic widgets from the example or polish it (using a real refinement list)

Reply:
Why not use the same as in the non-SSR example?